kratos/includes/model_part.h

```diff
@@ -1347,6 +1528,8 @@ class KRATOS_API(KRATOS_CORE) ModelPart : public DataValueContainer, public Flag
 
     SubModelPartsContainerType mSubModelParts;
 
+    MasterSlaveConstraintContainerType mMasterSlaveConstraints;
+
```

Comment:
You may move this to the mesh

Reply:
This means all the interface functions should also be added to the mesh class ?